app/exporters/publishing_api_draft_section_exporter.rb

```diff
@@ -1,6 +1,6 @@
 class PublishingApiDraftSectionExporter
   def call(section, manual)
-    organisation = OrganisationFetcher.fetch(manual.organisation_slug)
+    organisation = OrganisationsAdapter.new.find(manual.organisation_slug)
```

Comment:
Would it be any clearer to have a class method `find` that instantiated an instance of `OrganisationsAdaptor`?